frontend/packages/text-editor/src/TextEditor.tsx

```diff
@@ -70,6 +70,12 @@ export const TextEditor = ({
     }
   };
   const handleSearchChange = (event: any) => setSearchQuery(event.target.value);
+  const initialSelectedLangCodes = JSON.parse(localStorage.getItem('selectedLanguages')) || [];
+  [selectedLangCodes, setSelectedLangCodes] = useState<LangCode[]>(initialSelectedLangCodes);
```

Comment:
It seems that a `const` is missing here, and when I tried to add it locally, I got an error since `selectedLangCodes` and `setSelectedLangCodes` are already passed as props.

```suggestion
  const [selectedLangCodes, setSelectedLangCodes] = useState<LangCode[]>(initialSelectedLangCodes);
```

One way to fix it is by moving all your code into the parent component (`app-development/features/textEditor/TextEditor.tsx`).
We could also remove/fix the old code that stored the active apps in the url (**?lang=nb**) as we now store it in the local storage
```
  const setSearchQuery = (search: string) => {
    const params: any = { lang: searchParams.get('lang') };
    if (search.length > 0) {
      params.search = search;
    }
    setSearchParams(params);
  };
  useEffect(() => {
    if (appLangCodes && !appLangCodes.includes(selectedLangCodes[0])) {
      setSelectedLangCodes([defaultLangCode]).then();
    }
  }, [appLangCodes, selectedLangCodes]); // eslint-disable-line react-hooks/exhaustive-deps
```

Reply:
done